tensorboard/backend/http_util.py

```diff
@@ -44,6 +44,8 @@
 _CSP_SCRIPT_DOMAINS_WHITELIST = []
 _CSP_SCRIPT_HASHES_STRICT_DYNAMIC = True
 _CSP_SCRIPT_SELF = False
+# numericjs (via projector) uses unsafe-eval :(.
+_CSP_SCRIPT_UNSAFE_EVAL = True
```

Comment:
Does this need to _default_ to true, or can the projector plugin set it
to true for its HTML only so that we don’t also allow it for the rest of
TensorBoard?

Reply:
No, but ideally, we want nearly all of these CSP fields to be configurable so I think it makes a lot more sense to think of the new API holistically. 

I am thinking of expanding asset_provider that plugin authors can implement as part of the plugin API but I am not yet sure how CSP part should be configured; do we take shasum as input or does TensorBoard core parse the document to shasum the script tags? 

Since this PR is not as time critical as I originally thought, I will wait for your reply.